samples/matter/common/src/bridge/bridge_manager.cpp

```diff
@@ -31,96 +31,141 @@ void BridgeManager::Init()
 				     false);
 }
 
-CHIP_ERROR BridgeManager::AddBridgedDevices(BridgedDevice *aDevice, BridgedDeviceDataProvider *aDataProvider)
+CHIP_ERROR BridgeManager::AddBridgedDevices(MatterBridgedDevice *devices[], BridgedDeviceDataProvider *dataProvider,
+					    uint8_t deviceListSize)
 {
-	aDataProvider->Init();
-	CHIP_ERROR err = AddDevices(aDevice, aDataProvider);
+	VerifyOrReturnError(deviceListSize <= kMaxDevicesPerProvider, CHIP_ERROR_NO_MEMORY);
```

Comment:
I think we should get rid of it, as the customer may want to use more devices per provider and it will not be convenient to make him modify the core component. I think at some point we will just test it well and put the recommended max number of supported devices to the documentation, as a limitation. For now it may stay, but I would be happy to have at least TODO comment saying it should be removed once we will be confident of our solution.

Reply:
Will remove it and add some doxygen for `AddBridgedDevices` method. I guess we should add docs for all APIs but most probably we have a task for that.